src/index.html

```diff
@@ -3,10 +3,14 @@
 <head>
     <meta charset="UTF-8">
     <title>S.W.E.A.T</title>
+    <link rel="stylesheet" href="main.css">
 </head>
 <body>
     <p>Hello World</p>
```

Comment:
Project has been set up and tested. Dummy content no longer needed, please delete code in lines 9 -12